verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -217,17 +212,27 @@ void IndexingFactsTreeExtractor::ExtractModuleHeader(
     return;
   }
 
-  Visit(*port_list);
+  bool has_propagated_type = false;
```

Comment:
Can you add some //comments to explain this section, with examples of input code?
Are you making any assumptions with this particular implementation?